packages/main/src/Toolbar.ts

```diff
@@ -516,7 +516,7 @@ class Toolbar extends UI5Element {
 			const abstractItem = this.getItemByID(refItemId);
 			const eventType = e.type;
 			const eventTypeNonPrefixed: string = e.type.replace("ui5-", "");
-			const prevented = !abstractItem?.fireEvent(eventTypeNonPrefixed, e.detail, true);
+			const prevented = !abstractItem?.fireEvent(eventTypeNonPrefixed, e.detail, true, false);
```

Comment:
I would call `stopImmediatePropagation` to the original event since we fire new custom element from the abstract item instead of stop bubbling. Also include test.